### PR TITLE
Visualization issue 43 plot f1 scores

### LINKED ISSUE #43: Plot f1-scores of several models
Plot the f1-scores of the models that we have created so far: NaiveBayes, SVM, RandomForest, Zero-Shot

### PR DESCRIPTION
**Pull request by dominikmn**
Final version for the stakeholder review.

Closes #43

### CODE REVIEW

**Review by chrsteck — APPROVED** [review]
all good

File: visualization_plot_f1_scores.py [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+
+# %%
+from transformers import pipeline
+import pandas as pd
+import numpy as np
+from tqdm import tqdm
+from utils import loading, feature_engineering, scoring
+from sklearn.metrics import precision_score, recall_score, f1_score
+import seaborn as sns
+import plotly.express as px
```
Comment (lima-tango) [inline visualization_plot_f1_scores.py:28]: you are not using express and could remove the import. Same with seaborn, tqdm , pipeline, pio, and plt.

**Review by lima-tango — APPROVED** [review]
Nice plotly plot!

One optional improvement: you could remove unused imports.

File: visualization_plot_f1_scores.py (hunk 1 above)
Comment (dominikmn) [reply visualization_plot_f1_scores.py:25]: Done

**Pull request merged**